src/lib/flow-2.0.0.js

```diff
@@ -0,0 +1,1358 @@
+/**
```

Comment:
Using the same version of flowjs as in voto5 to make sure both frontend & backend are compatible